index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="description" content="Sample Microverse Hello Project">
+        <meta name="author" content="Albert Mensah-Ansah">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="style.css">
+        <title>My Portfolio</title>
+    </head>
+    <body>
+
+        <nav class="topnav">
+            <div class="welcome">Welcome</div>
+            <div>
+                <i class="fa fa-bars hamburgar"></i>
+            </div>
+        </nav>
+
+        <div class="container">
+            <div class="headline">
+                Hey There. I'm Albert
+                <div class="sub-headline">
+                    I'm a Software Developer
+                </div>
+            </div>
+            
+            <div class="supporting-text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </div>
+
+            <div >
+                <div class="social-media-icons">
+                    <img class ="icon" src="images/twitter.svg" alt="twitter">
+                    <img class ="icon" src="images/Linkedin_icon.svg" alt="LinkedIn">
+                    <img class ="icon" src="images/medium.svg" alt="Medium">
+                    <img class ="icon" src="images/github.svg" alt="github">
+                    <img class ="icon" src="images/angellist.svg" alt="angellist">
+                </div>  
+            </div>
```

Comment:
Kindly this is a list so it's more semantic to use `ul` tag to group them instead of `div`, and `li` per icon, and each icon has a link (`a` tag).
Like
```
<ul>
     <li><a href=".."><img  ...></a></li>
      ......
</ul>
```